File: Javascript/C_Strings/Exercicio_1272.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
z = parseInt(lines[0])
for(let i = 1;i<=z;i++){
    var a = lines[i].split(RegExp(" +"))
    s = ""
    for(var o of a){
        if(o){
            s+=o[0]
        }
    }
    console.log(s)
}

File: Python/C_Strings/Exercicio_1272.py
import re
leia = int(input())
lista = list()
codigo = ""
for x in range(leia):
    j = []
    codigo = ""
    frase = input()
    j = re.split(" +",frase)
    for z in range(len(j)):
        string = j[z]
        if string!="":
            codigo = codigo + string[0]
    print(codigo)
